src/Logging/src/Abstractions/DynamicLoggerProviderBase.cs

```diff
@@ -333,4 +333,12 @@ private IEnumerable<string> GetKeyPrefixes(string name)
 
         return null;
     }
+
```

Comment:
```suggestion
/// <inheritdoc cref="ISupportExternalScope"/>
```

Reply:
Since it's public it helps to have some docs even though its only ever called by framework.

Reply:
Agreed. Turns out just adding it without `cref` works as well:

![image](https://user-images.githubusercontent.com/104792814/206719768-d3e89be6-a837-40ac-895a-5618551215c8.png)